tests/flow/promises/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -587,12 +587,10 @@ Promise.reject(0)
   });
 
 // resolvedPromise<T> -> catch() -> then():?T
-Promise.resolve(0)
-  .catch(function(err) {})
-  .then(function(num) {
-    var a: ?number = num;
-    var b: string = num; // Error: string ~> number
-  });
+Promise.resolve(0).catch(function(err) {}).then(function(num) {
```

Comment:
I think that you should also check the body of the method. I don't think that we want to inline this.

Reply:
Yeah I’m not sure why it decided to do that. Do you happen to know what the `path` is that’s passed to `printMemberChain`? Is it the whole expression (`Promise.resolve(0).catch(...).then(...)`) or is it just the `Promise.resolve(0)`?